python/valis/db/queries.py

```diff
@@ -172,6 +172,28 @@ def get_targets_by_sdss_id(sdss_id: int) -> peewee.ModelSelect:
 
     return vizdb.SDSSidStacked.select().where(vizdb.SDSSidStacked.sdss_id == sdss_id)
 
+def get_targets_by_sdss_id_list(sdss_id_list: list = []) -> peewee.ModelSelect:
+    """ Perform a search for SDSS targets on vizdb.SDSSidStacked based on a list if sdss_ids.
+
+    Perform a search for SDSS targets from a list of sdss_id values, 
+    using the peewee ORM in the vizdb.SDSSidStacked table. 
+    We return the peewee ModelSelect directly here so it can be 
+    easily combined with other queries, if needed.
+
+    In the route endpoint itself, remember to return wrap this in a list.
+
+    Parameters
+    ----------
+    sdss_id_list : List[int]
+        list of sdss_id values
+
+    Returns
+    -------
+    peewee.ModelSelect
+        the ORM query
+    """
+
+    return vizdb.SDSSidStacked.select().where(vizdb.SDSSidStacked.sdss_id.in_(sdss_id_list))
```

Comment:
This function isn't that different than the existing `get_targets_by_sdss_id`.  Can you consolidate these two into a single function.